.github/workflows/ci.yaml

```diff
@@ -462,22 +483,32 @@ jobs:
         include:
           - cassandra-version: 2.1.22
             cucumber-options: '--tags ~@cassandra_4_0_onwards --tags @sidecar --tags ~@cassandra_3_11_onwards'
+            experimental: false
           - cassandra-version: 2.2.19
             cucumber-options: '--tags ~@cassandra_4_0_onwards --tags @sidecar --tags ~@cassandra_3_11_onwards'
-          - cassandra-version: 3.0.24
+            experimental: false
+          - cassandra-version: 3.0.25
             cucumber-options: '--tags ~@cassandra_4_0_onwards --tags @sidecar --tags ~@cassandra_3_11_onwards'
-          - cassandra-version: 3.11.10
+            experimental: false
+          - cassandra-version: 3.11.11
             cucumber-options: '--tags ~@cassandra_4_0_onwards --tags @sidecar'
+            experimental: false
+          - cassandra-version: 4.0.0
+            cucumber-options: '--tags @sidecar'
+            experimental: false
           - cassandra-version: 'github:apache/trunk'
             cucumber-options: '--tags @sidecar'
-        # sidecar test including postgres should only run with C* 3.11.10, so we exclude all other
+            experimental: true
+        # sidecar test including postgres should only run with C* 3.11.11, so we exclude all other
         exclude:
           - storage-type: postgresql
             cassandra-version: 2.1.22
           - storage-type: postgresql
             cassandra-version: 2.2.19
           - storage-type: postgresql
-            cassandra-version: 3.0.24
+            cassandra-version: 3.0.25
+          - storage-type: postgresql
+            cassandra-version: 3.11.11
```

Comment:
Do we want to exclude 3.11 here and only run against 4.0.0?

Reply:
It's the case actually, on this line we're excluding the 3.11.11 run for the postgresql backend. What will remain is 4.0.0 only.

Reply:
:+1:

Reply:
Sorry, I misread.
Yes, that's intentional because we're already testing all version with the Cassandra backend, and there's no value in testing them all with the postgres backend.